### PR TITLE
Adding three Samples for Laws of Gestalt

### PR DESCRIPTION
**Pull request by edgalindo**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

**Comment by edgalindo** [comment]
Hey Fabian hope to get all this right. Having lots of problems here with WiFi, but trying to get it done:-)

**Comment by fabianmoronzirfas** [comment]
I'll review it when I'm back in Germany 🇩🇪

**Comment by fabianmoronzirfas** [comment]
I @edgalindo This PR has to many changes. You changed files you should not have touched. And I can't review this. Please create a new fork und your account. Are you working on windows now? I saw some line break errors. Also your thumbnails are not in the size I requested and the large image is not called out.png

File: grid/p5js/chance_grid/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 layout: p5js
 author: Edmundo Mejía Galindo
-author-url: [https://github/edgalindo..........xxxxxxxx //ACHTUNG!!!: ÄNDERN!
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/grid/p5js/chance_grid
 title: Grid
 tags: P5.js, "Law of Proximity", "Law of Connected-Elements", "Law of Proximity", "Multistable-perception" Chance, Grid
```
Comment (fabianmoronzirfas) [inline grid/p5js/chance_grid/index.md]: Tags need to be in an array [P5.js, "Law of Proximity", "Law of Connected-Elements", "Law of Proximity", "Multistable-perception" Chance, Grid]

File: README.md [hunk 2]
```diff
@@ -34,6 +34,23 @@ What does it means to design with code? Is it a way of thinking? Is it a tool? I
 
 I don't know where this will take me but there might be 🐉 and 🐛.  
 
+<<<<<<< HEAD
```
Comment (fabianmoronzirfas) [inline README.md]: This is from having and conflict. I think you are working with a not up to date fork of the repo.

File: README.md [hunk 3]
```diff
@@ -34,6 +34,23 @@ What does it means to design with code? Is it a way of thinking? Is it a tool? I
...
+- form
+- - 2D
+- - 3D
+- grid
+- interaction
+- iteration
+- motion
+- typography
+
+>>>>>>> fabianmoronzirfas/cube/edit
```
Comment (fabianmoronzirfas) [inline README.md]: This is also a conflict marker

File: chance/p5js/chance_symetry/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/chance/p5js/chance_symetry
+title: Symetry
+tags: P5.js, "Law of Connected-Elements", "Law of Regularity", "Law of Symmetry", Chance, Symmetry
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/index.md]: Needs to be in an array

File: chance/p5js/chance_symetry/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/chance/p5js/chance_symetry
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/index.md]: This points to something that is not a author url. A author url should be something like your homepage or a your github account page

File: chance/p5js/chance_symetry/index.md (hunk 4 above)
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/index.md]: needs to be an array   

    [P5.js, "Law of Connected Elements", "Law of Regularity", "Law of Symmetry", Chance, Symmetry]

File: chance/p5js/chance_symetry/index.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+title: Symetry
+tags: P5.js, "Law of Connected-Elements", "Law of Regularity", "Law of Symmetry", Chance, Symmetry
+permalink: /symetry/Symetry/
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/index.md]: There is no category symetry

File: chance/p5js/chance_symetry/index.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+summary: This sample shows a static grid regular distances between the arcs.
+---
+
+The static grid is built with arc lines that are placed in two different directions. The viewer has the feeling that the grid moves like a vertigo and makes it visually at the same time dynamic.
+
+![](http://www2.pic-upload.de/img/32318239/symetry.jpg)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/index.md]: No hardcoded URLs please. The p5js layout adds the working code to the page

File: chance/p5js/chance_symetry/sketch.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
+var a = 0;
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/sketch.js]: please use meaningful names for variables

File: chance/p5js/chance_symetry/sketch.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+function setup() {
+  createCanvas(500, 500);
+  background(255);
+  
+}
+
+function draw() {
+  noLoop();
+  smooth();
+    for(var d =10; d < 500; d += 10){
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance_symetry/sketch.js]: no magic numbers. Please use width

File: form/3d/basiljs/cube/cube.jsx [hunk 10]
```diff
@@ -1 +1,5 @@
...
  //b.popMatrix();
  // ----------
  var fname = File($.fileName).parent.fsName + '/' + ($.fileName.split('/')[$.fileName.split('/').length - 1]).split('.')[0] + '.indd';
  // b.println(fname);
  doc.save(fname, false, 'basil', true);
  b.savePNG('out.png');
}
b.go();
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (fabianmoronzirfas) [inline form/3d/basiljs/cube/cube.jsx]: Something went totally wrong with this file. You shouldn't even have touched this nor committed it.

File: grid/p5js/chance_grid/index.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/grid/p5js/chance_grid
+title: Grid
+tags: P5.js, "Law of Proximity", "Law of Connected-Elements", "Law of Proximity", "Multistable-perception" Chance, Grid
```
Comment (fabianmoronzirfas) [inline grid/p5js/chance_grid/index.md]: Array. As before. Also the Law Of Connected Elements is written without a `-`

File: grid/p5js/chance_grid/index.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+permalink: /grid/Grid/
```
Comment (fabianmoronzirfas) [inline grid/p5js/chance_grid/index.md]: Grid is a bit general name. I'm sure you can come up with something better

File: grid/p5js/chance_grid/index.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+summary: This sample shows a dynamic grid, that changes the width and height of itself.  
+---
+
+The grid changes the size of the rectangles that are built through the grid each time by clicking the updating button. It is a non symetric drawn horizontal and vertical line canvas, that makes it interesting for drawing sheet.  
+
+![](http://www2.pic-upload.de/img/32318238/grid.jpg)
```
Comment (fabianmoronzirfas) [inline grid/p5js/chance_grid/index.md]: No hard coded URLS please. The canvas from the P5js sketch is added dynamically to the page when using the p5js layout

File: interact/p5js/interaction_contrast/index.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/interact/p5js/interaction_contrast
+title: Contrast
+tags: P5.js, "Law of Closure", "Law of Connected-Elements", "Law of Contrast", "Law of Figure-Ground", "Law of Regularity", "Law of Similarity", "Law of Simplicity", "Law of Symmetry", Interaction, Contrast
```
Comment (fabianmoronzirfas) [inline interact/p5js/interaction_contrast/index.md]: Array and so on…

File: interact/p5js/interaction_contrast/index.md [hunk 15]
```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/interact/p5js/interaction_contrast
```
Comment (fabianmoronzirfas) [inline interact/p5js/interaction_contrast/index.md]: Does not point to an account

File: grid/p5js/chance_grid/sketch.js [hunk 16]
```diff
@@ -0,0 +1,16 @@
+function setup() {
+  createCanvas(500, 500);
+  background(255);
+  
+  for(var x = random(500); x > 0.1; x /= 1.1) {
```
Comment (fabianmoronzirfas) [inline grid/p5js/chance_grid/sketch.js]: Please do it dynamic. Add `width`

File: interact/p5js/interaction_contrast/index.md [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+title: Contrast
+tags: P5.js, "Law of Closure", "Law of Connected-Elements", "Law of Contrast", "Law of Figure-Ground", "Law of Regularity", "Law of Similarity", "Law of Simplicity", "Law of Symmetry", Interaction, Contrast
+permalink: /contrast/Contrast/
```
Comment (fabianmoronzirfas) [inline interact/p5js/interaction_contrast/index.md]: There is no category contrast

File: interact/p5js/interaction_contrast/sketch.js [hunk 18]
```diff
@@ -0,0 +1,26 @@
+  var x, y, a, b;
```
Comment (fabianmoronzirfas) [inline interact/p5js/interaction_contrast/sketch.js]: use meaningful names

**Pull request closed without merging**